core/sqf/sql/scripts/sqconfig

```diff
@@ -20,7 +20,7 @@
 # @@@ END COPYRIGHT @@@
 
 begin node
-_virtualnodes 2
+_virtualnodes 4
 end node
```

Comment:
Intentional to change to 4 virtual nodes for single-nodes? Or was this just for testing?